src/main/java/duke/exceptions/DukeException1.java

```diff
@@ -0,0 +1,30 @@
+package duke.exceptions;
+
+import java.lang.Exception;
+
+/**
+ * Exception that is thrown when there are errors encountered in the Duke application.
+ */
+public class DukeException1 extends Exception {
```

Comment:
Same comment here as above. Try to avoid naming classes/variables with numbers (DukeException1) at the back, as stated here, under W4.6d.